docker/bash_prompt

```diff
@@ -0,0 +1,99 @@
+#!/bin/bash
```

Comment:
Under which license is that file?

I am thinking that this is going to far. convenience for interactive usage should be in another image that is added on top and not included by default.